component/list/list.css

```diff
@@ -0,0 +1,57 @@
+.drawer-list{
+    display: inline-block;
+    grid-area: d;
+    color: var(--black);
+    background-color: var(--white);
+    overflow: auto;
+    border-radius: 5px;
+}
+.drawer-unordered-list{
```

Comment:
In this class, you could have added display flex property. 
The result would be that
1. You would not have to define drawer-list-item as inline-block (line no 16)
2. You could have reused this unordered class in the stack list too and there you would only have to change the flex-direction to the column.

Reply:
Thanks for suggestion Anshul! I will fix it